apps/storefront/src/pages/Dashboard/index.tsx

```diff
@@ -128,6 +141,18 @@ function Dashboard(props: PageProps) {
     });
   };
 
+  const onStartMasquerade = (companyId: number) => {
+    try {
+      const cartEntityId = z.string().parse(Cookies.get('cartId'));
```

Comment:
I feel like I explained myself poorly here. `Cookies.get` is already returning `string|undefined`, no need to parse that. I was expecting the value to be more volatile (`string|number|object|undefined|null`... which might grant the parsing)